src/ProjectTemplates/Web.ProjectTemplates/content/BlazorWeb-CSharp/BlazorWeb-CSharp/Identity/Extensions/RedirectNavigationManagerExtensions.cs

```diff
@@ -0,0 +1,38 @@
+using System.Diagnostics.CodeAnalysis;
+
+namespace Microsoft.AspNetCore.Components;
+
+internal static class RedirectNavigationManagerExtensions
```

Comment:
Another thought: Maybe we should move these to another namespace so they're not as visible to the customer?